requirements.txt

```diff
@@ -37,8 +37,11 @@ SpeechRecognition==3.1.3
 telepot==4.1
 
 # Text To Speech Module
-pygame==1.9.2b6
+pygame>=1.9.2b6
 pyvona==1.1
 
 # Creating Sphinx Language Model
-poster==0.8.1
\ No newline at end of file
+poster==0.8.1
+
+# Weather
```

Comment:
when i move it back to travis script, pywapi can't be found. see here https://travis-ci.org/rachmadaniHaryono/Melissa-Core/builds/166953481 for this commit https://github.com/rachmadaniHaryono/Melissa-Core/commit/6679e8003a79037f0ee9db4a48ade156dccc77b2